tests/07-dnsthread.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Goal of this test:
+# - ensure that dnsquery can find simple addresses
+
+### Constants
+addr_output="${s_basename}-addrs.txt"
+
+
+### Actual command
+scenario_cmd() {
+	# Check address
+	setup_check_variables
+	(
+		${dnsthread_resolve} localhost:80 > ${addr_output}
```

Comment:
Maybe this should test a variety of lookups?  Raw IP addresses, names which need to actually get looked up...?

Reply:
I've added some more test cases, but I avoided names that actually need to get looked up -- that would require an active internet connection, and (so far) none of the tests require one.

That said, I suppose that I could support it with the same general idea as the setuidgid test -- if it fails due to the relevant errno, I could output a special code to indicate "skip this test".  I'd rather have that as a separate `tests/??-*.sh` filename, though.  Like, `07-dnsthread-local.sh` and `08-dnsthread-network.sh`.